templates/download/desktop/contribute.html

```diff
@@ -15,10 +15,6 @@
 <div id="contribute-introductions" class="row row-hero contribution-intro-row">
     <div class="strip-inner-wrapper">
         <div class="contribution-intro nine-col">
-            <h1>Tell us what we should do more&hellip;</h1>
-            <p>&hellip;and put your money where your mouth is ;)</p>
-        </div>
-        <div class="contribution-intro nine-col">
             <h1>Help shape the future of Ubuntu&hellip;</h1>
             <p>&hellip;tell us what to do more of!</p>
```

Comment:
Do you think we could find a way to remove the "of!"? Maybe simply remove it and leave "Help shape the future of Ubuntu"? Or make the second line "What should we focus on?" or "Tell us what's most important to you."